src/NServiceBus.Core/Receiving/ReceiveComponent.cs

```diff
@@ -242,6 +260,31 @@ void AddReceivers(IBuilder builder, RecoverabilityExecutorFactory recoverability
             var pushSettings = new PushSettings(configuration.LocalAddress, errorQueue, configuration.PurgeOnStartup, requiredTransactionSupport);
             var dequeueLimitations = configuration.PushRuntimeSettings;
 
+            var hasOwningSatellite = false;
+            foreach (var satellitePipeline in configuration.SatelliteDefinitions)
+            {
+                var owningSatelliteDefinition = satellitePipeline as OwningSatelliteDefinition;
```

Comment:
Probably this code could be made a bit cleverer. Didn't invest too much time on it